controllers/clusteroperator_test.go

```diff
@@ -45,7 +45,10 @@ func TestUpdateCOStatusDisabled(t *testing.T) {
 	reconciler.OSClient = fakeconfigclientset.NewSimpleClientset(co)
 
 	for _, tc := range tCases {
-		reconciler.updateCOStatusDisabled()
+		err := reconciler.updateCOStatusDisabled()
```

Comment:
Is this linting related? If not, we should not worry about it in this PR.

Reply:
yes, it forces you to check errors